src/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -203,7 +204,20 @@ public boolean onPreferenceClick(Preference preference) {
             
         }
         
-       
+        if (BuildConfig.DEBUG) {
```

Comment:
This option must be enable on navigation_drawer (DEBUG mode) instead of the settings screen.